spec.md

```diff
@@ -245,6 +243,34 @@ querying of the Service Broker's catalog, the Platform might not have an
 end-user with which to associate the request, therefore in those cases the
 originating identity header would not be included in those messages.
 
+## Service Broker Errors
+
+When a request to a Service Broker fails, the Service Broker MUST return an
+appropriate HTTP response code. If an appropriate response code is not defined
+within this specification, Service Broker authors SHOULD choose a code that
```

Comment:
This SHOULD bothers me because of the MUST in the first sentence. SHOULD implies something is optional, so what's the alternative to "choosing a code that accurately reflects the nature of the failure"? Choosing a random value? The MUST prevents that.

Perhaps just:
- - - - 
When a request to a Service Broker fails, the Service Broker MUST return an
appropriate HTTP response code. When the specification defines the expected
response code that value MUST be used.
- - - - 


Not sure we need to say anything more.

Reply:
Good spot, thanks @duglin! Updated so need your LGTM again.